modules/mono/editor/bindings_generator.cpp

```diff
@@ -2244,9 +2279,9 @@ Error BindingsGenerator::_generate_cs_signal(const BindingsGenerator::TypeInterf
 		p_output.append(INDENT1 "[DebuggerBrowsable(DebuggerBrowsableState.Never)]" MEMBER_BEGIN
 								"private static readonly StringName " CS_STATIC_FIELD_SIGNAL_NAME_PREFIX);
 		p_output.append(p_isignal.name);
-		p_output.append(" = \"");
-		p_output.append(p_isignal.name);
-		p_output.append("\";\n");
+		p_output.append(" = SignalName.");
+		p_output.append(p_isignal.proxy_name);
+		p_output.append(";\n");
```

Comment:
Same as the method name field. This field should be removed, and the usages should be made to use the new one.